src/probnum/prob/distributions/normal.py

```diff
@@ -375,14 +448,15 @@ def sample(self, size=()):
             loc=self.mean(), scale=self.std(), size=size, random_state=self.random_state
         )
 
-    def reshape(self, newshape):
-        if np.prod(newshape) != 1:
-            raise ValueError(
-                f"Cannot reshape distribution with shape {self.shape} into shape {newshape}."
-            )
-        self.parameters["mean"] = np.reshape(self.parameters["mean"], newshape=newshape)
-        self.parameters["cov"] = np.reshape(self.parameters["cov"], newshape=newshape)
-        self._shape = newshape
+    def transpose(self, *axes):
+        return Normal(
+            mean=self.mean().copy(),
+            cov=self.cov().copy(),
+            random_state=self.random_state,
+        )
+
+    def _reshape_inplace(self, newshape):
+        raise NotImplementedError
```

Comment:
Inplace reshaping for univariate normals is actually impossible due to the fact that the attributes of numpy's array scalars are immutable. See also https://numpy.org/doc/stable/reference/arrays.scalars.html. We should overthink #130. It would be best if scalar quantities behave exactly the same as arrays. Maybe we should use arrays of shape () internally?

Reply:
@JonathanWenger

Reply:
Yes i think representing everything internally as arrays makes a lot of sense. This removes quite a few annoying conditionals in the code as well.

Reply:
Alright, I'll rephrase #130 then?